mobx_lint/README.md

```diff
@@ -0,0 +1,52 @@
+mobx_lint is a developer tool for users of mobx, designed to help stop common issues and simplify repetitive tasks.
```

Comment:
I think we should mention this in the docs as well so folks can see it easily on the website ... perhaps it can be a separate PR